edx_django_utils/plugins/docs/how_tos/how_to_create_a_plugin_app.rst

```diff
@@ -32,12 +32,13 @@ file::
 
 3. (optional, but recommended) Create a top-level settings/ directory with common.py and production.py modules. This will allow you to use the PluginSettings.CONFIG option as written below.
 
-4. configure the Plugin App in their AppConfig
+4. configure the Plugin App in their AppConfig. Note that in this example, we are explicitly configuring plugins for use in edx-platform. If your plugin is going to be used in another IDA, you may have different project and settings types. You will need to look at the IDA in question for what values it expects. You may want to add new values to the relevant enums.
+
 class::
 
    from django.apps import AppConfig
    from edx_django_utils.plugins.constants import (
-       ProjectType, SettingsType, PluginURLs, PluginSettings, PluginContexts
+       ProjectType, PluginURLs, PluginSettings, PluginContexts
```

Comment:
Actually, if we are going to leave this as an LMS example, why not leave `ProjectType` and `SettingsType` as-is, except moving them to a separate import that is edx-platform specific. You might even move your note to be a code comment against the platform specific import statement. Thoughts?

Reply:
I'm not sure we want to encourage importing from edx-platform

Reply:
Yes. I forgot this is plugin code. I was confused by `ProjectType` being left as-is. Should that also be converted to strings?

Reply:
Also, I wonder if the ultimate solution is to just move all the constants to the library? What is the down side of that? Not sure if we'd need LMSSettingsType, etc. Anyway, I know that affects scope of this, but it is also difficult to use the example code if we leave a bunch of strings and it isn't clear where they are coming from.

Reply:
The downside that I can see is the precedent of having IDA-specific code in plugins that should theoretically be able to go in any IDA. I don't think we'd want to add constants every time we enabled plugins in a new IDA that had its own configuration vars.

Reply:
For now I'd just as soon remove ProjectType, although it actually appears in a few more places in the documentation, so I should remove it there too.

Reply:
Sure. Maybe just add a code comment next to the first of the strings pointing to `ProjectType` so someone knows where they are coming from?

Reply:
Doesn't `ProjectType` also need to be dropped from the import?